# coding: utf-8

from the_tale.game.map.places.exceptions import PlacesException

class PlaceModifierException(PlacesException): pass
